### ISSUE TITLE
Marker to place a cursor at a specified place after replacement insertion

### ISSUE THREAD

**Issue by cpkio** [jfcherng/Sublime-TypeShort#3]
Hi, thanks for this great plugin, helps a lot.

Could you add a feature of an indicator inside the replacement string, where the cursor should be placed after insertion? Something like "fj_": "" or anytihing, so the cursor would be at "%!"?

**Comment by jfcherng** [comment]
Sorry for late reply. I accidentally did not see this issue somehow.

This feature would be good in some situation.
I would like to implement this in the next release. :+1:

**Comment by jfcherng** [comment]
Implemented in `1.4.0` via b418f17f03dd2fd06b311296f70daefd742646cd.

I choose to use `{|}` to represent the new cursor position but it could be configured with `cursor_placeholder` if people don't like it.

```javascript
{
    // the symbol used to represent the new cursor position after a replacement
    "cursor_placeholder": "{|}",

    "bindings": [
        {
            // convert 'cmt ' into '<!--  -->' and place the cursor at its mid
            "keymaps": {
                "cmt ": "<!-- {|} -->"
            },
            "syntax_list": ["text.html", "text.html.basic"]
        }
    ]
}
```
